src/page-modules/departures/server/journey-planner/index.ts

```diff
@@ -358,3 +383,54 @@ type RecursivePartial<T> = {
     ? RecursivePartial<T[P]>
     : T[P];
 };
+
+const mapToMapLegs = (
+  pointsOnLink: GraphQlPointsOnLink | undefined,
+  transportMode: TransportModeType,
+  transportSubmode: TransportSubmodeType | undefined,
+  fromStopPlace?: { id: string; latitude?: number; longitude?: number },
+) => {
+  if (!pointsOnLink || !pointsOnLink.points) return [];
+  const points = polyline.decode(pointsOnLink.points);
+  const fromCoordinates: [number, number] = [
+    fromStopPlace?.latitude || 0,
+    fromStopPlace?.longitude || 0,
+  ];
```

Comment:
I prefer using nullish coalescing in cases like these (`??`). Not sure what's the best convention however, probably doesn't matter.

Reply:
In this case, I guess `??` is more correct since `fromStopPlace.longitude` is either a number or undefined 🤔 I'll update.